tests/IntegrationTest.php

```diff
@@ -41,7 +41,7 @@ protected static function getFixturesDir()
      */
     protected static function getTempFile()
     {
-        return self::getFixturesDir().\DIRECTORY_SEPARATOR.'.tmp.php';
+        return tempnam(sys_get_temp_dir(), 'PHP-CS-Fixer');
```

Comment:
From php doc: `Only the first 63 characters of the prefix are used. Windows even uses only the first three characters of the prefix.`

better to not use `tempnam` but use some rand suffix.

Reply:
So, Linux will take full prefix, Windows will take `PHP` as prefix. yet, it's only the prefix, file will be unique.
Where's the issue of using `tempnam`?